SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-intro.md

```diff
@@ -19,21 +19,33 @@ Following this tutorial, you implement cross-server remote procedure calls (RPCs
 _Image: Example of cross-server shooting using multiserver zoning functionality._ </br></br>
 
 The tutorial demonstrates that the workflows and iteration speed you’re used to as an Unreal developer are almost entirely unaltered by the GDK: it’s just like regular Unreal.
-</br>
-</br>
-</br>
-**Notes:**
 
-* Before starting this tutorial you need to complete the [Example Project set up guide]({{urlRoot}}/content/get-started/example-project/exampleproject-intro). 
+### Change your branch of the Example Project 
+
+Before starting this tutorial you need to complete the [Example Project set up guide]({{urlRoot}}/content/get-started/example-project/exampleproject-intro). Following this, you clone or download and set up all the elements of the GDK and its dependencies, including the Example Project.
+
+To follow the Multiserver zoning shooter tutorial, you need to change your Example Project branch from `release` (which is the default) to `multiserver-tutorial-start`. To do this:
+
+1. If you have Unreal Editor open, close it.
+2. In GitHub, change your Example Project branch to `multiserver-tutorial-start`.</br>
+To do this, open a terminal window, navigate to directory containing your cloned Example Project  - this will be    `...\UnrealEngine\samples\UnrealGDKExampleProject` and enter `git checkout -b multiserver-tutorial-start`.
+3. Open the project in Visual Studio.</br>
+To do this, still in your terminal window, in the directory `...\UnrealEngine\samples\UnrealGDKExampleProject\Game`, double-click on `GDKShooter.sln`.
```

Comment:
they don't need to be in a terminal window to open up an sln file.

Reply:
Ta - it's just from Windows Explorer?